packages/react/src/lib/factories.ts

```diff
@@ -91,6 +92,10 @@ type CreateShorthandFactoryConfigInner<TPropName = string> = {
   mappedArrayProp?: TPropName
 }
 export type CreateShorthandFactoryConfig = CreateShorthandFactoryConfigInner
+export type CreateShorthandFactoryResult = (
+  a: ShorthandValue<UIComponentProps>,
+  b: CreateShorthandOptions,
```

Comment:
```suggestion
  options?: CreateShorthandOptions,
```

This param is also optional